### PR TITLE
Adding the way to add the SCP for ADFS

### PR DESCRIPTION
**Pull request by j0rt3g4**
Followed the recommendation to add the PS Code to add the service connection Point. Issue #3053

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md [hunk 1]
```diff
@@ -354,12 +355,37 @@ Sign-in the AD FS server with domain administrator equivalent credentials.
...
+
 ### Enrollment Agent Certificate Enrollment
 
 Active Directory Federation Server used for Windows Hello for Business certificate enrollment perform their own certificate lifecycle management.  Once the registration authority is configured with the proper certificate template, the AD FS server attempts to enroll the certificate on the first certificate request or when the service first starts.
 
 Approximately 60 days prior to enrollment agent certificate’s expiration, the AD FS service attempts to renew the certificate until it is successful.  If the certificate fails to renew, and the certificate expires, the AD FS server will request a new enrollment agent certificate.  You can view the AD FS event logs to determine the status of the enrollment agent certificate.
 
+
+### Service Connection Point (SCP) in Active Directory for ADFS Device Registration Service
+Now you will add the Service connection Point to ADFS device registration Service for your Active directory by running the following script:
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md]: Add the Service connection Point to ADFS device registration Service for your Active directory by running the following script:

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md [hunk 2]
```diff
@@ -354,12 +355,37 @@ Sign-in the AD FS server with domain administrator equivalent credentials.
...
+
+>[!TIP] Make sure to change the $enrollmentService and $configNC variables before running the script.
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md]: >[!TIP] 
> Make sure to change the $enrollmentService and $configNC variables before running the script.

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md [hunk 3]
```diff
@@ -354,12 +355,37 @@ Sign-in the AD FS server with domain administrator equivalent credentials.
...
+ 
+$de = New-Object System.DirectoryServices.DirectoryEntry
+$de.Path = "LDAP://CN=Device Registration Configuration,CN=Services," + $configNC
+
+$deSCP = $de.Children.Add("CN=62a0ff2e-97b9-4513-943f-0d221bd30080", "serviceConnectionPoint")
+$deSCP.Properties["keywords"].Add("enterpriseDrsName:" + $enrollmentService)
+$deSCP.CommitChanges()
+```
+
+>[!NOTE] You can save the modified script in notepad and save them as "add-scpadfs.ps1" and the way to run it is just navigating into the script path folder and running .\add-scpAdfs.ps1.
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md]: >[!NOTE] 
> You can **paste** the modified script in notepad and save **it** as "add-scpadfs.ps1" To run it, navigate into the script path folder and run ".\add-scpAdfs.ps1".

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md [hunk 4]
```diff
@@ -354,12 +355,37 @@ Sign-in the AD FS server with domain administrator equivalent credentials.
...
+>
+
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md]: clean up formatting

**Comment by JohanFreelancer9** [comment]
Copy edited. No further changes.

**Comment by mypil** [comment]
@j0rt3g4  - please merge the commit on the copy/edit review made by nenonix or make the necessary changes on the PR following the suggestions she made. Thank you.

**Comment by mypil** [comment]
@DaniHalfin - can you please assist to check, approve and merge the content for this PR based on issue #3053?

Thank you.

**Comment by j0rt3g4** [comment]
Hi @mypil I think I've merge all the content into the pr. Please let me know if it's something missing guys

**Comment by j0rt3g4** [comment]
Hi @mypil I think I've merge all the content into the pr. Please let me know if it's something missing guys

**Comment by DaniHalfin** [comment]
@mapalko - can you take a look ?

**Comment by illfated** [comment]
@j0rt3g4 : there are still some redundant blank lines in your PR source:
260, 358, 365, 388.
Also: Please add a NewLine/Break between `>[!NOTE]` and `You can save the modified script in notepad [ ... ]`
as well as for `>[!TIP] Make sure to change the $enrollmentService and $configNC variables before running the script.`

To me, it looks like you were a bit quick to mark the conversations as Resolved, before any changes have been applied.

**Comment by j0rt3g4** [comment]
@illfated  Fixed that :)

**Comment by j0rt3g4** [comment]
Probably what I did was the other way, (Resolved) thought they were applied when I marked them as resolved thinking that the changes were gonna be applied automatically. Guess not

**Comment by mypil** [comment]
@mapalko @DaniHalfin - Can you please assist to check, approve and merge the content for this PR based on issue #3053?

Thank you.

**Comment by mypil** [comment]
@mapalko @DaniHalfin - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #3053?

Thank you.

**Comment by mypil** [comment]
@mapalko @DaniHalfin - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #3053? Thank you.

cc: @Dansimp

**Comment by illfated** [comment]
Judging by what @mapalko said earlier in a different PR, I think he lacks the merge rights to the repository. (I don't know if this has been verified yet.)

**Comment by mypil** [comment]
@AndreaBarr - would you be able to merge the content of this PR? Thank you.

**Pull request merged**